### ISSUE TITLE
G signal

### ISSUE THREAD

**Issue by SargTeX** [project-error/npwd#1111]
### Development Issue

Yes

### Phone Version

3.8.1-beta.0"

### Issue Description

How can i make g signal work for ambulance job and receive message if someone press G signal in death screen

### Issue Reproduction

4

**Comment by SargTeX** [comment]
i using original esx_ambulancejob

**Comment by itschip** [comment]
You can look at this for reference: https://github.com/project-error/npwd/blob/master/examples/qbcore/ambulance_alert.lua

**Comment by SargTeX** [comment]
i need for esx this to make it work

**Comment by itschip** [comment]
Yes, but the logic is the same. Just swap out what's qbcore to esx
